packages/core/src/App/Containers/Layout/header/traders-hub-header.tsx

```diff
@@ -155,7 +156,7 @@ const TradersHubHeader = observer(() => {
                     {is_real_acc_signup_on && <RealAccountSignup />}
                 </React.Fragment>
             )}
-            {is_mobile && (
+            {!isDesktop && (
```

Comment:
Here and in line 116 you are checking `isDesktop`  twice, can ternary be used? Please, ignore if irrelevant